cmd/minishift/cmd/addon/addon_enable.go

```diff
@@ -38,7 +38,7 @@ var addonsEnableCmd = &cobra.Command{
 }
 
 func init() {
-	addonsEnableCmd.Flags().IntVar(&priority, priorityFlag, 0, "The priority of the add-on when it is applied.")
+	addonsEnableCmd.Flags().IntVar(&priority, priorityFlag, 0, "The priority of the add-on. Determines the order in which to apply the add-on.")
```

Comment:
A single priority cannot determine the order of a single add-on. More than one add-ons are needed to establish an order.